src/app/components/Layout/Footer/coponents/WrapperFooter.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const WrapperFooter = styled.footer`
+	display: flex;	
+	justify-content: center;
+	align-items: center;
+    position: absolute;
```

Comment:
Will see how it'll work with the content of the page. Absolutely positioned footer will overlap it.